ci-operator/step-registry/single-node/conf/single-node-conf-commands.sh

```diff
@@ -26,8 +27,13 @@ if not "controlPlane" in cfg:
 cfg["controlPlane"]["replicas"] = 1
 
 # Single Node Openshift requires extra memory and compute resources
-assert "aws" in cfg["controlPlane"]["platform"], "Only AWS single-node is supported for now"
-cfg["controlPlane"]["platform"]["aws"]["type"] = "'${SINGLE_NODE_AWS_INSTANCE_TYPE}'"
+platform = cfg["controlPlane"]["platform"]
+if "aws" in platform:
+    platform["aws"]["type"] = "'${SINGLE_NODE_AWS_INSTANCE_TYPE}'"
+elif "gcp" in platform:
```

Comment:
A switch statement on platform is an anti-pattern in steps. Instead there should be individual config steps per platform.

This is just a suggestion I would encourage you to consider. I won't block my approval in GCP on something in single-node. So let me know if you disagree and I can approve.

Reply:
Done